app/UserPreferences.m

```diff
@@ -61,6 +66,9 @@ - (void)setFontSize:(NSNumber *)fontSize {
     [_defaults setObject:fontSize forKey:kPreferenceFontSizeKey];
 }
 
+- (NSNumber *)identifier {
+    return self.theme.identifier;
+}
```

Comment:
Introducing a new identifier user defaults key that then has to live forever is a bad idea, in particular because I eventually want users to be able to create their own themes.

Reply:
In my latest commit I have changed these to just use the theme name. Do you think that's any better? Before the app was just using the background and foreground colours to identify each theme (so if two themes had the same colour scheme they would appear to be the same).